package.json

```diff
@@ -42,7 +46,23 @@
     "sinon": "^2.1.0"
   },
   "scripts": {
-    "test": "mocha --require coffee-script/register --compilers coffee:coffee-script --compilers js:babel-core/register test/*.js",
-    "watch:test": "npm t -- --watch"
+    "test": "xo --quiet && mocha --require coffee-script/register --compilers coffee:coffee-script --compilers js:babel-core/register test/*.js",
+    "watch:test": "npm t -- --watch",
+    "lint": "xo --quiet"
+  },
+  "xo": {
+    "space": true,
+    "semicolon": false,
+    "rules": {
+      "no-var": "warn",
```

Comment:
why warn? why do we need `var`? let it error in case there is `var` in code

Reply:
I used this configuration from the `datahub-cli`

Reply:
But, this is not `datahub-cli` - let's not use var if there is no need for that

Reply:
@zelima will be fixed